pkg/test/resourcefixture/testdata/basic/bigqueryconnection/v1alpha1/bigqueryconnectionconnection/cloudsqlconnectionbasic/_http.log

```diff
@@ -1188,7 +1188,7 @@ X-Xss-Protection: 0
 
 {
   "cloudSql": {
-    "database": "sqldatabase-sample-${uniqueId}",
+    "database": "projects/${projectId}/instances/sqlinstance-sample-${uniqueId}/databases/sqldatabase-sample-${uniqueId}",
```

Comment:
This is unexpected. I don't think this works with real GCP API? My understanding is that the underlying GCP API expects the name of the database.

Reply:
According to the API doc,  this field is just the name of db